code/datums/components/crafting/equipment.dm

```diff
@@ -83,6 +83,17 @@
 	time = 5 SECONDS
 	category = CAT_EQUIPMENT
 
+/datum/crafting_recipe/barbeque_grill
```

Comment:
Now that the grill is craftable, you should really lower the price of the cargo pack, and make it more appealing to boot somehow.

Reply:
Any ideas on what the new price should be? I'll let you decide

Reply:
I could also add some ready made wood, coal and other fuel sources to the cargo pack to make it more appealing

Reply:
Halve the price, and do that, yeah.

Reply:
Ok so it already comes with fuel & tongs, just adjusted price from 1600cr to 800cr